modules/core/src/v2/wallets.ts

```diff
@@ -265,18 +276,37 @@ export class Wallets {
    * @param params.gasPrice
    * @param params.disableKRSEmail
    * @param params.walletVersion
+   * @param params.multisigType optional multisig type, 'onchain' or 'tss'; if absent, we will defer to the coin's default type
    * @returns {*}
    */
   async generateWallet(params: GenerateWalletOptions = {}): Promise<WalletWithKeychains> {
     common.validateParams(params, ['label'], ['passphrase', 'userKey', 'backupXpub']);
     if (!_.isString(params.label)) {
       throw new Error('missing required string parameter label');
     }
+
+    const isTss = params.multisigType ? params.multisigType === 'tss' : this.baseCoin.supportsTss();
```

Comment:
can this just be const isTss = params.multisigType === 'tss'

also, should multisigType be an enum? 

```
enum MultiSigType {
  TSS = 'tss',
  ON_CHAIN = 'onchain',
}

multisigType?: MultiSigType
```

Reply:
good idea. ill update to enum.

Reply:
also, this statement doesn't work:
`const isTss = params.multisigType === MultiSigType.TSS || this.baseCoin.supportsTss();`

because if user provides `multisigType: 'onchain'`, we should still try to create the onchain wallet, and not the TSS wallet. this will also allow us to continue developing against single sig wallets for Sol / Dot.

Reply:
ah okay - I was misinterpreting the ternary

Reply:
on second thought, i think i'll leave this as is unless there's a strong case to switch to Enum.
from a usability perspective, it is more convenient to be able to pass in a string to `multisigType`:
```
await wallets.generateWallet({
        label: 'tss wallet',
        passphrase: 'passphrase',
        multisigType: 'tss',
      }
```

as opposed to referring to the actual enum:
```
await wallets.generateWallet({
        label: 'tss wallet',
        passphrase: 'passphrase',
        multisigType: MultisigEnum.TSS,
      }
```